docs/languages/swift.md

```diff
@@ -0,0 +1,30 @@
+# Swift
+
+## Install Syntax Highlighting
+```vim
+:TSInstall swift
+```
+
+## Install Language Server
+
+```vim
+:LspInstall sourcekit
+```
+
+
+## Configure Lunarvim
+
+The Lsp, and formatting does not work out of the box. To fix this, do the following:
+
+### Lsp Configuration
+Create a file called `~/.config/lvim/ftplugin/swift.lua`:
+
+```lua
+local opts = {}
```

Comment:
Do swift projects usually include c/c++ files? what about a small mention of the `root_dir` pattern? 

https://github.com/neovim/nvim-lspconfig/blob/ec7119b166b16e681f663fcbf16b7139b38172ae/lua/lspconfig/server_configurations/sourcekit.lua#L4-L8

Reply:
I think `Sourcekit` can be used as a replacement to other C-based langs LSP's.
From the [SourceKit-LSP](https://github.com/apple/sourcekit-lsp): 
<img width="688" alt="image" src="https://user-images.githubusercontent.com/48220549/155298621-793de62f-91ed-4119-99ef-ea9da468ec34.png">
Adding mention about root_dir pattern.

Reply:
Does commit 86ffafefbdcab5a1e7652596341c25dda42c4e92 fix this, or should I add some more info?

Reply:
@Uzaaft, it depends, I'm still wondering if clangd needs to be disabled when enabling this one. Please start the sourcekit server (probably open a swift file), then open a c/c++ file and see what happens and how it plays along with clangd.

Reply:
It's mainly just to see if they both show up in `:LspInfo`.  Does go-to definition and other LSP features still work for these files? any apparent conflict errors happening? That sort of thing. If it still doesn't seem to be broken, then it probably isn't 😄

Reply:
They both show up, and no issues when I jumped from a swift file to a cpp file inside a hello world project and the swift compiler project